source/administration/config-settings.rst

```diff
@@ -112,6 +123,42 @@ Enable Team Directory ``"EnableTeamListing": false``
 
 ________
 
+Policy (Enterprise)
+``````````````````````````
+Settings to configure the permission restrictions for sending team invite links and managing channels.  
+
+Enable sending team invites from ``"RestrictTeamInvite": "all"``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Restrict the permission levels required to send team invites. Note: if permissions are restricted and "Get Team Invite Link" is used to share a link, it will need to be regenerated after the desired users joined the team.
+
+``all``: Selecting "All team members" allows any team member to invite others using an email invitation or team invite link.
+
+``admin``: Selecting "Team and System Admins" hides the email invitation and team invite link in the Main Menu from users who are not Team or System Admins.
+
+``system_admin``: Selecting "System Admins" hides the email invitation and team invite link in the Main Menu from users who are not System Admins.
+
+Enable public channel management permissions for ``"RestrictPublicChannelManagement": "all"``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Restrict the permission levels required to to create, delete, rename, and set the header or purpose for public channels. The last member of a public channel has the ability to archive the channel regardless of their permission level.
+
+``all``: Selecting "All team members" allows any team member to create, delete, rename, and set the header or purpose for public channels.
+
+``admin``: Selecting "Team and System Admins" restricts channel management permissions for public channels to Team and System Admins.
+
+``system_admin``: Selecting "System Admins" restricts channel management permissions for public channels to System Admins.
```

Comment:
Propose

``system_admin``: Channel management permissions for public channels restricted to System Admins.